src/components/KeyDataAssets/KeyDataAssets.tsx

```diff
@@ -23,64 +25,42 @@ const KeyDataAssets = () => {
         ))}
       </div>
       <h3 className="mt-9 text-center text-[#292929] font-medium text-[36px] lg:text-[48px]">
-        Enable teams to find{" "}
-        <span className="text-[#7147E8]">key data assets</span>
+        {currentUsecase?.header.split(" ").slice(0, -3).join(" ")}{" "}
+        <span className="text-[#7147E8]">
+          {currentUsecase?.header.split(" ").slice(-3).join(" ")}
+        </span>
```

Comment:
@sweta1308 , what are we trying to achieve here?

Reply:
Updated